http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/RuntimeServiceInterpSpec.scala

```diff
@@ -819,34 +821,65 @@ class RuntimeServiceInterpTest
   }
 
   it should "get a runtime" in isolatedDbTest {
+    val mockSamService: SamService[IO] = mock[SamService[IO]](defaultMockitoAnswer[IO])
+    when(
+      mockSamService.checkAuthorized(
+        isEq(userInfo.accessToken.token),
+        any(runtimeSamResource.getClass),
+        isEq(RuntimeAction.GetRuntimeStatus)
+      )(any(Ask[IO, AppContext].getClass))
+    ).thenReturn(IO.unit)
+    val service = makeRuntimeService(samService = mockSamService)
     val res = for {
       samResource <- IO(RuntimeSamResourceId(UUID.randomUUID.toString))
       testRuntime <- IO(makeCluster(1).copy(samResource = samResource).save())
-      getResponse <- runtimeService.getRuntime(userInfo, testRuntime.cloudContext, testRuntime.runtimeName)
+      getResponse <- service.getRuntime(userInfo, testRuntime.cloudContext, testRuntime.runtimeName)
     } yield getResponse.samResource shouldBe testRuntime.samResource
     res.unsafeRunSync()(cats.effect.unsafe.IORuntime.global)
   }
 
-  it should "throw ClusterNotFoundException for nonexistent clusters" in isolatedDbTest {
-    val exc = runtimeService
-      .getRuntime(userInfo, CloudContext.Gcp(GoogleProject("nonexistent")), RuntimeName("cluster"))
+  it should "throw RuntimeNotFoundException when user has no access" in isolatedDbTest {
```

Comment:
I'm not sure this test is hitting `SamService.checkAuthorized`.. seems it is throwing 404 first because the runtime doesn't exist?